developer-docs-site/docs/tutorials/validator-node/connect-to-testnet.md

```diff
@@ -35,3 +38,34 @@ Only do this if you got confirmation email from Aptos team for your eligibility.
         --from-file=validator-identity.yaml=validator-identity.yaml \
         --from-file=validator-full-node-identity.yaml=validator-full-node-identity.yaml
     ```
+
+## Verify node connections
```

Comment:
maybe link to the node liveness doc? https://aptos.dev/reference/node-liveness-criteria/#verifying-the-liveness-of-your-node cc @rajkaramchedu for how to organize this info into a centralized place